index.js

```diff
@@ -0,0 +1,49 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let select = document.querySelector("select");
+  let div = document.querySelector("#movieInfo");
+  let form = document.querySelector("form");
+  let submissions = document.querySelector("ul");
+  let input = document.querySelector("#review");
+  let title = document.createElement("h3");
+
+  const movieOptions = async () => {
+    try {
+      let res = await axios.get(`https://ghibliapi.herokuapp.com/films`);
+      res.data.forEach(event => {
```

Comment:
a more descriptive variable name for `event` could have been `movie`. event is usually a keyword when manipulating the DOM.